xml/book_admin.xml

```diff
@@ -12,7 +12,7 @@
 <!-- Administration Guide                                                  -->
 <!-- ##################################################################### -->
 <?provo dirname="admin/"?>
-<book version="5.0" xml:lang="en" xml:id="book-sle-admin"
+<book version="5.0" xml:lang="en" xml:id="book-admin"
```

Comment:
A complete word:

```suggestion
<book version="5.0" xml:lang="en" xml:id="book-administration"
```

(The suggestion is not consistent with install/upgrade now, because we use the noun and not the verb here. But `book-administrate` sounds off.)